packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -58,11 +55,35 @@ import {
   getMediaType,
   getFullMimeWhitelist,
 } from '../message-utils'
-
 import { normalizeWebchatSettings } from '../components/webchat-settings'
+import { getProperty } from '../util/objects'
 
-import merge from 'lodash.merge'
-import { useAsyncEffect } from 'use-async-effect'
+export const getParsedAction = botonicAction => {
+  const splittedAction = botonicAction.split('create_case:')
+  if (splittedAction.length <= 1) return undefined
+  return JSON.parse(splittedAction[1])
+}
+
+/*
+ * returns the value of a property defined in bot's theme based on WEBCHAT.CUSTOM_PROPERTIES dictionary.
+ * it gives preference to nested defined properties (e.g.: header.style) over plain properties (e.g.: headerStyle).
+ * if property doesn't exist, returns the defaultValue
+ */
+export const _getThemeProperty = theme => (
```

Comment:
no export. No _

Reply:
export because it is used for tests, _ to distinguish it from local scoped getThemeProperty